src/api/lib/backend/test.rb

```diff
@@ -0,0 +1,47 @@
+# API for accessing to the backend
+module Backend
+  class Test
+    @backend = nil
+
+    # Starts the test backend
+    def self.start(wait_for_scheduler = false)
```

Comment:
*nitpick*: I would rather use named parameters for this...

`Backend::Test.start(true)` vs. `Backend::Test.start(wait_for_scheduler: true)`